releasenotes/notes/36655.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: telemetry
+releaseNotes:
+  - |
+    **Updated** Implement `Telemetry.AccessLog.Filter`.
```

Comment:
I'm thinking this is an **Added** and not an updated implementation (but maybe I'm wrong). If it is added function, then maybe something like `**Added Common Expression Language (CEL) filter support.`

Also add the issue, I think https://github.com/istio/istio/issues/36514.